File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        for (int x =0; x<nums.length-1; x++){
            for(int y = nums.length-1; y>0; y--){
                if (nums[x] == nums[y]){
                    if(x==y){
                        continue;
                    }
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-1.py
class Solution:
    def findMin(self, nums: List[int]) -> int:
        #re-sort the list
        l = 0 
        r = len(nums)-1 
        res = 0 
        while nums[l] > nums[r] and l<r:
            nums.append(nums[res])
            l += 1
            res += 1
        
        #binary search for smallest
        return nums[res]